evaluation/analysis.py

```
@@ -10,161 +10,174 @@
+    ]].groupby('model', as_index=False).mean().set_index('model')
+
+    architecture_eval_scores['overall'] = architecture_eval_scores[
+        ['conversation_relevancy', 'conversation_completeness']
+    ].mean(axis=1)
+    architecture_eval_scores = architecture_eval_scores.sort_values(
+        by='overall',
+        ascending=False
+    )[['overall']]
+
+    # make pretty table
+    ARCHITECTURE_EVAL_TABLE = architecture_eval_scores.style \
```

Ruff (0.8.2)

Local variable `ARCHITECTURE_EVAL_TABLE` is assigned to but never used

Remove assignment to unused variable `ARCHITECTURE_EVAL_TABLE`

(F841)